src/status_im/translations/en.cljs

```diff
@@ -135,22 +135,22 @@
    :backup-your-seed                     "Backup your Seed Phrase"
 
    ;;seed
-   :your-data-belongs-to-you             "Your Data and Funds belong to you. We can’t help get it back if you loose it"
-   :your-data-belongs-to-you-description "This means Status can’t help you get it back if you lose it. You are in charge of the secuty of all your data"
+   :your-data-belongs-to-you             "If you lose your seed phrase you lose your data and funds."
```

Comment:
Haha, classic dev mindset. The marketing people will have a frothy if we ship aggressive tips 😉 

`You're fully in control of all your data, so keep your seed phrase as safe as possible!`

Reply:
Agree it is direct, but think it conveys the importance of backing up.  We don't want to be too relaxed here when it comes to data/account loss.
Also checked with @hesterbruikman and got 👍 .

Added a new line "backing up your seed phrase is the best safeguard" since we don't actually talk about backing up on the backup screen 🤣